active/NumberBox/NumberBox.md

```diff
@@ -80,6 +94,10 @@ XAML
 
 ### Add increment and decrement stepping
 
+Use the `SpinButtonPlacementMode` property to enable buttons in the NumberBox control that can be clicked to increment or decrement the value in the NumberBox. This defaults to `Hidden`, set to `Inline` to enable the buttons. The amount of increment/decrement is specified with the `StepFrequency` property, which defaults to 1.
+
+<!-- todo: What happens if I set this as well as AcceptsCalculation? Is this ignored? -->
```

Comment:
They work together -- you could enter "5 + 3", hit enter to get "8", and then press up or down to get 9 or 7. Although I don't know what happens if you type in an expression and press one of the arrows before evaluation occurs... I added a note for myself to check that in #1505.

Reply:
Clicking the button will trigger a focus change, maybe that will evaluate it.

Reply:
Changing focus should trigger the evaluation. The Spinner button press should then effect the newly evaluated value. For example, when the NumberBox contains "2 + 2" as input, and the "Up" Spinner button is pressed, the final value should read "5" as "2+2=4" is evaluated before the increment occurs.

This behavior was evaluated and approved at an earlier spec review.

Reply:
The spinner buttons don't get focus. I verified that currently, it ignores everything you typed, which is not so good. :) I will handle it as a bug.